pkg/quota/evaluator/core/pods.go

```diff
@@ -115,10 +121,32 @@ func (p *podEvaluator) Constraints(required []api.ResourceName, item runtime.Obj
 	requiredSet := quota.ToSet(required)
 	missingSet := sets.NewString()
 	for i := range pod.Spec.Containers {
-		enforcePodContainerConstraints(&pod.Spec.Containers[i], requiredSet, missingSet)
+		if len(pod.Spec.Containers[i].VolumeMounts) == 0 {
```

Comment:
Yes, but we also need podVolume info, so add pod to the parameters
Done, thanks